storage_scrubber/Cargo.toml

```diff
@@ -36,9 +36,18 @@ rustls-native-certs.workspace = true
 once_cell.workspace = true
 
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
-chrono = { workspace = true, default-features = false, features = ["clock", "serde"] }
-reqwest = { workspace = true, default-features = false, features = ["rustls-tls", "json"] }
-aws-config = { workspace = true, default-features = false, features = ["rustls", "sso"] }
+chrono = { workspace = true, default-features = false, features = [
+    "clock",
+    "serde",
+] }
+reqwest = { workspace = true, default-features = false, features = [
```

Comment:
can you undo these changes for the three crates? They are unrelated and make it less readable IMO.